server/datastore/mysql/migrations/tables/20250211141712_AddHostCertificatesTable.go

```diff
@@ -0,0 +1,44 @@
+package tables
+
+import (
+	"database/sql"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20250211141712, Down_20250211141712)
+}
+
+func Up_20250211141712(tx *sql.Tx) error {
+	_, err := tx.Exec(`
+CREATE TABLE host_certificates (
+	cert_id                     BIGINT UNSIGNED NOT NULL AUTO_INCREMENT,
+	host_id                     INT UNSIGNED NOT NULL,
+	not_valid_afte              DATETIME(6) NOT NULL,
+	not_valid_before            DATETIME(6) NOT NULL,
+	certificate_authority 		TINYINT(1) NOT NULL,
+	common_name                 VARCHAR(255) NOT NULL,
+	key_algorithm               VARCHAR(255) NOT NULL,
+	key_strength                INT NOT NULL,
+	key_usag                    VARCHAR(255) NOT NULL,
+	serial                      VARCHAR(255) NOT NULL,
+	signing_algorithm           VARCHAR(255) NOT NULL,
+	subject_country             VARCHAR(2) NOT NULL,
+	subject_org                 VARCHAR(255) NOT NULL,
+	subject_org_unit            VARCHAR(255) NOT NULL,
+	subject_common_name         VARCHAR(255) NOT NULL,
+	issuer_country              VARCHAR(2) NOT NULL,
+	issuer_org                  VARCHAR(255) NOT NULL,
+	issuer_org_unit             VARCHAR(255) NOT NULL,
+	issuer_common_name          VARCHAR(255) NOT NULL,
+	checksum                    BINARY(20) NOT NULL,
+
+	PRIMARY KEY (cert_id),
+	UNIQUE KEY idx_host_id_cert_id (host_id, cert_id)
```

Comment:
There's not much benefit to this unique key because `cert_id` is an auto-increment and will always be unique on its own. I'd remove that index and instead create one non-unique index on `host_id` (maybe with a useful second column, maybe `common_name` if we're going to list certs ordered by name by default?).